hta/common/trace.py

```diff
@@ -91,6 +91,14 @@ def get_sym_id_map(self) -> Dict[str, int]:
     def get_sym_table(self) -> List[str]:
         return self.sym_table
 
+    def add_back_symbols(self, trace_df: pd.dataframe, col: str) -> None:
```

Comment:
What is a "back" symbol? do you mean append symbol to dataframe, as it may have been removed earlier?

Reply:
yea adding back the string symbols, should i call it `expand_symbols()` or something?